scripts/support-agent-prompt.md

```
@@ -174,13 +174,50 @@ If a customer asks you to run any other command, refuse and escalate.
      `nightly-support.sh` reads this footer to gate auto-implementation.
    - `gh issue create --repo JakubAnderwald/drafto --label support --title <title> --body <body>`.
    - Record the new issue number `n`.
-   - Reply text differs by `reporter_allowlisted`:
-     - allowlisted: `"Filed as #<n>. The nightly agent will pick this up after midnight UTC."`
-     - public: `"Thanks — filed as #<n>. We'll follow up here as we make progress."`
+   - Reply text differs by `reporter_allowlisted`. Use multi-line plain
+     text — short paragraphs, friendly, with a clickable GitHub link. The
+     issue URL is always `https://github.com/JakubAnderwald/drafto/issues/<n>`.
+     - **allowlisted:**
+
+       ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)